src/features/auth/lib/authMachine.ts

```diff
@@ -746,6 +757,9 @@ export const authMachine = createMachine<
       }),
       clearSession: () => removeSession(wallet.myAccount as string),
       deleteFarmIdUrl: deleteFarmUrl,
+      setTransactionId: assign<Context, any>({
```

Comment:
Could we potentially clear the `transactionId` upon `entry` into the `authorised` state so we don't have a stale value hanging around in context?